web/src/message_list.js

```diff
@@ -414,6 +417,9 @@ export class MessageList {
         $recipient_row.find(".topic_edit_form").empty();
         $recipient_row.find(".topic_edit").hide();
         $recipient_row.find(".always_visible_topic_edit").show();
+        $recipient_row.find(".on_hover_topic_resolve").show();
+        $recipient_row.find(".on_hover_topic_unresolve").show();
+        $recipient_row.find(".zulip-icon-mute").show();
```

Comment:
Read `recipient_row.hbs` more carefully; we should be hiding/showing `on_hover_topic_mute` and `on_hover_topic_unmute`, not only `zulip-icon-mute`. The `on_hover_` names are more consistent with the other things we're showing/hiding here, but more importantly, your code is broken if the topic is already muted.

Reply:
Yeah, this needs a similar fix for mute/unmuted case like `resolve / unresolve`.  My bad for not reviewing more carefully.